tests/Gaufrette/Functional/Adapter/PhpseclibSftpTest.php

```diff
@@ -3,7 +3,44 @@
 namespace Gaufrette\Functional\Adapter;
 
 use Gaufrette\Adapter\PhpseclibSftp;
+use Gaufrette\Filesystem;
+use phpseclib\Net\SFTP;
 
 class PhpseclibSftpTest extends FunctionalTestCase
 {
+    /** @var SFTP */
+    private $sftp;
+
+    /** @var string */
+    private $baseDir;
+
+    public function setUp()
+    {
+        $host     = getenv('SFTP_HOST');
+        $port     = getenv('SFTP_PORT') ?: 22;
+        $user     = getenv('SFTP_USER');
+        $password = getenv('SFTP_PASSWORD');
+        $baseDir  = getenv('SFTP_BASE_DIR');
+
+        if ($host === false || $user === false || $password === false || $baseDir === false) {
```

Comment:
That's a long debate :p But generally speaking I'm against it: (strict) equality is the only comparison operator for which yoda conditions might make sense. Thus it introduces more oddities than anything else. And in the meantime, I think what could be solved by yoda conditions should be solved by code review.